src/content/en/2020/resource-hints.md

```diff
@@ -231,47 +263,47 @@ Implementing a `dns-prefetch` fallback in the same `<link>` tag causes a [bug](h
 Close to 2% of pages (~40k) reported the issue of both `preconnect` & `dns-prefetch` in a single resource.
 
 {{ figure_markup(
-  caption="Pages that pass the preconnect Lighthouse audit",
+  caption="Pages that pass the `preconnect` Lighthouse audit.",
   content="19.67%",
   classes="big-number",
   sheets_gid="152449420",
   sql_file="lighthouse_preconnect.sql"
 ) }}
 
-We saw only 19.67% of pages passing Lighthouse’s "[Preconnect to required origins](https://web.dev/uses-rel-preconnect/)" audit, creating a large opportunity for thousands of websites to start using `preconnect` or `dns-prefetch` to establish early connections to important third-party origins.
+We saw only 19.67% of pages passing Lighthouse's "[Preconnect to required origins](https://web.dev/uses-rel-preconnect/)" audit, creating a large opportunity for thousands of websites to start using `preconnect` or `dns-prefetch` to establish early connections to important third-party origins.
 
 {{ figure_markup(
-  caption="Pages that pass the preload Lighthouse audit",
+  caption="Pages that pass the `preload` Lighthouse audit.",
   content="84.6%",
   classes="big-number",
   sheets_gid="1047875076",
   sql_file="lighthouse_preload.sql"
 ) }}
 
-Running Lighthouse’s "[Preload key requests](https://web.dev/uses-rel-preload/)" audit resulted in 84.6% of pages passing the test, which is an astonishing result. If you are looking to use `preload` for the first time, remember, fonts and critical scripts are a good place to start.
+Running Lighthouse's "[Preload key requests](https://web.dev/uses-rel-preload/)" audit resulted in 84.6% of pages passing the test. If you are looking to use `preload` for the first time, remember, fonts and critical scripts are a good place to start.
 
-{# TODO - revisit this sentence - Ref https://github.com/HTTPArchive/almanac.httparchive.org/pull/1587#discussion_r532291496 #}
+{# TODO(authors/reviewers) - revisit this sentence - Ref https://github.com/HTTPArchive/almanac.httparchive.org/pull/1587#discussion_r532291496 #}
```

Comment:
I removed the part where we say `which is an astonishing result`, but @bazzadp related to your comment https://github.com/HTTPArchive/almanac.httparchive.org/pull/1587#discussion_r532291496, I am not sure at this point how to surface this information. How you think is best we share it?